### PR TITLE
JEN-1084 do not allow setting/changing parameters for trunk jobs

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: jenkins/trunk.yml [hunk 1]
```diff
@@ -17,115 +18,10 @@
...
-        name: MTR_REPEAT
-        default: "1"
-        description: Run each test N number of times, --repeat=N
-    - choice:
-        name: LABEL
-        choices:
-        - docker
-        - docker-32gb
-        description: Run build on specified instance type
+        description: "Do not change the value of this patameter. This is a trunk job and it should not be run manually."
```
Comment (laurynas-biveinis) [inline jenkins/trunk.yml]:
```suggestion
        description: "Do not change the value of this parameter. This is a trunk job."
```

File: jenkins/trunk.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@
     project-type: matrix
     defaults: global
     description: |
+        Do not run this job manually!
```
Comment (laurynas-biveinis) [inline jenkins/trunk.yml]: It's OK to run trunk jobs manually, it's not OK to change params
```suggestion
```

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: jenkins/trunk.yml (hunk 2 above)
Comment (hors) [reply jenkins/trunk.yml]: Thank you, fixed

File: jenkins/trunk.yml (hunk 1 above)
Comment (hors) [reply jenkins/trunk.yml]: I have corrected it.

**Review by delgod — APPROVED** [review]

**Pull request merged**